lib/mods/replace-get-set.js

```diff
@@ -24,16 +24,37 @@ function isDataProperty(type, propertyName) {
       dataProperties.get(type).includes(propertyName);
 }
 
+/**
+ * @param {Node} node - the current ts-morph AST Node
+ *
+ * @return {boolean} whether the node is a global get
+ */
+function isGlobalGet(node) {
+  return nodeTypes.isPropertyAccessExpression(node) &&
+    !nodeTypes.isCallExpression(node.getParent()) &&
+    isGlobal(node.getExpression());
+}
+
+/**
+ * @param {Node} node - the current ts-morph AST Node
+ *
+ * @return {boolean} whether the node is a global set
+ */
+function isGlobalSet(node) {
+  return nodeTypes.isBinaryExpression(node) &&
+    node.getOperatorToken().getText() === '=' &&
+    nodeTypes.isPropertyAccessExpression(node.getLeft()) &&
+    isGlobal(node.getLeft().getExpression());
+}
+
 /**
  * Replaces getset calls in the current node with the Get-Originals version
  *
  * @param {Node} node - a ts-morph AST Node
  * @param {Object} imports - the current list of imports
  */
 function replaceGlobalGetSetCalls(node, imports) {
-  if (nodeTypes.isPropertyAccessExpression(node) &&
-      !nodeTypes.isCallExpression(node.getParent()) &&
-      isGlobal(node.getExpression())) {
+  if (isGlobalGet(node)) {
```

Comment:
After this refactoring, there is a lot of duplicated code between the two branches. Is it possible to reduce that? We could do it in a separate PR if you'd prefer.